packages/@aws-cdk/aws-codepipeline-actions/test/servicecatalog/test.servicecatalog-action.ts

```diff
@@ -0,0 +1,9 @@
+import { expect, haveResourceLike } from '@aws-cdk/assert';
+import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import { App, Stack } from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+import * as cpactions from '../../lib';
+
+export = {
+
+};
```

Comment:
Take a look at our existing tests in this package for inspiration ([example](https://github.com/aws/aws-cdk/blob/f99c3271ed2b4c68f3cd2970a1b38571f5ddc911/packages/%40aws-cdk/aws-codepipeline-actions/test/bitbucket/test.bitbucket-source-action.ts#L12-L50)).